### PR TITLE
CORE-5831: Add custom serilizers to object mapper

### PR DESCRIPTION
**Pull request by yift-r3**

### CODE REVIEW

**Comment by yift-r3** [comment]
@driessamyn - FYI

File: libs/http-rpc/json-serialization/src/test/kotlin/net/corda/common/json/serialization/JsonSerializationTest.kt [hunk 1]
```diff
@@ -17,9 +18,9 @@ class JsonSerializationTest{
         val df = SimpleDateFormat("dd-MM-yyyy hh:mm")
         df.timeZone = TimeZone.getTimeZone("UTC")
         val date: Date = df.parse("01-01-1970 01:00")
-        val event = Event("party", date.toInstant())
+        val event = Event("party", date.toInstant(), MemberX500Name.Companion.parse("O=Alice, L=London, C=GB"))
```
Comment (charlieR3) [inline libs/http-rpc/json-serialization/src/test/kotlin/net/corda/common/json/serialization/JsonSerializationTest.kt:21]: Not picky, but the test name doesn't cover the change to this test. I.e. the test name is about date/instance but it's also checking member x500 name now.

**Review by charlieR3 — DISMISSED** [review]
LGTM

File: libs/http-rpc/json-serialization/src/test/kotlin/net/corda/common/json/serialization/JsonSerializationTest.kt (hunk 1 above)
Comment (LizR3) [reply libs/http-rpc/json-serialization/src/test/kotlin/net/corda/common/json/serialization/JsonSerializationTest.kt:21]: Agreed; it's kind of hidden here. With a rename of the test this looks great.

**Review by charlieR3 — APPROVED** [review]

**Review by LizR3 — APPROVED** [review]

**Review by ronanbrowne — APPROVED** [review]
LGTM (build changes)

**Review by vkolomeyko — APPROVED** [review]
LGTM (on behalf of @corda/http-rpc)

**Pull request merged**